apps/src/redux/unitSelectionRedux.js

```diff
@@ -119,7 +116,6 @@ export const asyncLoadCoursesWithProgress = () => (dispatch, getState) => {
 // Initial state of unitSelectionRedux
 const initialState = {
   scriptId: null,
-  unitName: null,
```

Comment:
This `unitName` was not updated whenever scriptId was updated and generally more confusing than it was worth. I took it out in favor of retrieving the unit by the unitId here.